### ISSUE TITLE
Feature: add array operators

### ISSUE THREAD

**Issue by afilp** [gid-oss/dataui-nestjs-crud#19]
Can you please please add the code of this PR into your package?

https://github.com/nestjsx/crud/pull/754

We really need to be able to query in an array!

Thanks a lot!

**Comment by zaro** [comment]
@afilp  I am willing to do it, please open a PR in this repo and I will take a look.

**Comment by afilp** [comment]
Thanks a lot! 

Here it is!
https://github.com/gid-oss/dataui-nestjs-crud/pull/21

**Comment by afilp** [comment]
Hi @zaro Did you have time to look at the PR? Thanks again!

**Comment by zaro** [comment]
@afilp  yes I did take a look, but I am having some strange problems running the tests locally so I haven't managed to figure out what exactly is causing the failure.

**Comment by zaro** [comment]
@afilp  this was released, so please open new issue if neede.
